base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -83,11 +83,14 @@ void OverrideInstanceJson(const selector::LocalInstanceGroup& group,
                         Json::Value& instance_json) {
   instance_json["instance_name"] = instance.name();
   instance_json["status"] = HumanFriendlyStateName(instance.state());
-  instance_json["adb_port"] = instance.adb_port();
+  if (instance.id() > 0) {
+    // These only make sense for active instances
+    instance_json["webrtc_device_id"] = instance.webrtc_device_id();
+    instance_json["adb_port"] = instance.adb_port();
+  }
```

Comment:
Can this be combined with the other `if (instance.id() > 0) {` block?

Reply:
Done